plugins/hdfs_assetstore/server/assetstore.py

```diff
@@ -101,11 +101,17 @@ def validateInfo(doc):
         return doc
 
     def capacityInfo(self):
-        info = self.client.df()
-        return {
-            'free': info['capacity'] - info['used'],
-            'total': info['capacity']
-        }
+        try:
+            info = self.client.df()
+            return {
+                'free': info['capacity'] - info['used'],
+                'total': info['capacity']
+            }
+        except:
```

Comment:
Can you change this to a more specific exception?  Even just `except Exception:` would be better (so it doesn't catch base exceptions).